stl/inc/format

```diff
@@ -314,6 +314,8 @@ public:
     using iterator       = const_iterator;
 
 private:
+    using _Size_type = typename basic_string_view<_CharT>::size_type;

```

Comment:
`basic_string_view::size_type` is mandated to be `size_t`; should that simply be hardcoded here?